root-packages/docker/build.sh

```diff
@@ -2,14 +2,14 @@ TERMUX_PKG_HOMEPAGE=https://docker.com
 TERMUX_PKG_DESCRIPTION="Set of products that use OS-level virtualization to deliver software in packages called containers."
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=20.10.14
+TERMUX_PKG_VERSION=20.10.15
 LIBNETWORK_COMMIT=9db86fb7b8d62a276cb784bf9c4520bf8187dd19
 DOCKER_GITCOMMIT=a224086
```

Comment:
Tag `v20.10.15` seems to be associated with commit `fd82621d35d2d9662854c8351976d5cc1e4186ce`.